xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -45,6 +45,23 @@ using namespace PLAYLIST;
 using namespace PVR;
 using namespace KODI::MESSAGING;
 
+namespace
+{
+  void AppendAudioStreamFlagsAsBooleans(CVariant& list, StreamFlags flags)
+  {
+    list["isdefault"] = ((flags & StreamFlags::FLAG_DEFAULT) != 0);
+    list["isoriginal"] = ((flags & StreamFlags::FLAG_ORIGINAL) != 0);
+    list["isimpaired"] = ((flags & StreamFlags::FLAG_VISUAL_IMPAIRED) != 0);
+  }
+
+  void AppendSubtitleStreamFlagsAsBooleans(CVariant& list, StreamFlags flags)
+  {
+    list["isdefault"] = ((flags & StreamFlags::FLAG_DEFAULT) != 0);
+    list["isforced"] = ((flags & StreamFlags::FLAG_FORCED) != 0);
+    list["isimpaired"] = ((flags & StreamFlags::FLAG_HEARING_IMPAIRED) != 0);
+  }
```

Comment:
> **Exception:** Do not indent namespaces to simplify nesting them and wrapping *.cpp* files in a namespace.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#32-indentation

Reply:
identation fixed, but i don't understand what means " wrapping .cpp files in a namespace." i'm not a C++ dev

Reply:
wrapping in a namespace means using an anonymous namespace so that symbols are mangled for the current file and don't conflict with static symbols in other files. In this case, it would look like:

```c++
namespace
{
  void AppendSubtitleStreamFlagsAsBooleans(CVariant& list, StreamFlags flags)
  {
    list["isdefault"] = ((flags & StreamFlags::FLAG_DEFAULT) != 0);
    list["isforced"] = ((flags & StreamFlags::FLAG_FORCED) != 0);
    list["isimpaired"] = ((flags & StreamFlags::FLAG_HEARING_IMPAIRED) != 0);
  }
}
```

Then `AppendSubtitleStreamFlagsAsBooleans` can be used freely in this file without conflicting with `AppendSubtitleStreamFlagsAsBooleans` defined in other files.